.travis.yml

```diff
@@ -1,19 +1,22 @@
 language: c
 env:
-  - OCAML=4.02.3
-  - OCAML=4.03.0
   - OCAML=4.04.2
   - OCAML=4.05.0
-  - OCAML=4.06.0
+  - OCAML=4.06.1
   - OCAML=4.07.0
 script:
-  - echo "yes" | sudo add-apt-repository ppa:avsm/ppa
+  # The opam package in this repository seems not to be working.
+  # - echo "yes" | sudo add-apt-repository ppa:avsm/ppa-opam-experimental
+  # - sudo apt-get install -qq opam
+  - echo "yes" | sudo add-apt-repository ppa:ansible/bubblewrap
```

Comment:
What is `bubblewrap` for?

Reply:
OPAM 2.0 requires it for sandboxing. It is possible to work without it, but then you need to pass an option to each `opam` call.

Reply:
I was wrong, it is enough to pass `--disable-sandboxing` to `opam init` for this package not to be needed.